src/styles/components/_presentation.scss

```diff
@@ -164,15 +160,15 @@
   text-overflow: ellipsis;
 }
 
-.presentation .thumbnail .caption .size {
+.presentation .caption .size {
```

Comment:
Yes I tested. There is no `.size` class outside of `.caption`, [link](https://github.com/strukturag/spreed-webrtc/blob/master/static/partials/presentation.html#L51).